src/modules/channels.js

```diff
@@ -2,11 +2,11 @@ const DEFAULT_CHANNEL = Symbol('DEFAULT_CHANNEL');
 
 class ChannelAlreadyExistsError extends Error {
   constructor(name, config) {
-    const message = `Channel "${name}" already exists with config ${JSON.stringify(config)}`;
+    const message = `Channel "${name.toString()}" already exists with config ${JSON.stringify(config)}`;
 
     super(message);
 
-    this.name = name;
+    this.name = name.toString();
```

Comment:
this is the only place you need to change, all other places no need to explicitly add `toString()` since it being converted automatically

Reply:
Symbols are not converting automatically, and that is the issue. It is a special behavior for the `Symbol` type.

If you try on node console:
```
> const x = Symbol("X")
undefined
> x.toString()
'Symbol(X)'
> "asd " + x.toString()
'asd Symbol(X)'
> "asd " + x
Uncaught TypeError: Cannot convert a Symbol value to a string
```